lib/chef/audit/audit_reporter.rb

```diff
@@ -138,9 +138,13 @@ def post_auditing_data
               return
             else
               # Save the audit report to local disk
-              error_file = "failed-audit-data.json"
+              error_file = "audit-data.json"
```

Comment:
can we just leave the reporting name the way it was and not change this "api"?  in production when you're likely to care, it really will be a "failed-audit-data.json" file and on chef-zero/zolo it won't matter and you won't care...